### ISSUE TITLE
Exception encountered

### ISSUE THREAD

**Issue by ThePaladinsForge** [Gjum/pyAgar.io#15]
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/gagar/window.py", line 121, in draw
    self.draw_subscriber.on_draw_cells(c, self)
  File "/usr/local/lib/python3.4/dist-packages/gagar/subscriber.py", line 29, in wrapper
    handler(*args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/gagar/subscriber.py", line 29, in wrapper
    handler(*args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/gagar/draw_cells.py", line 47, in on_draw_cells
    split_for = now - self.split_times[cell.cid]
KeyError: -1
```

**Comment by Gjum** [comment]
fixed in 607848e69c2dffc980d1eab0ff43c38d88878867

**Comment by Gjum** [comment]
`split_times` should really be stored in each cell instance using a custom `Cell` class
